checkstyle-tester/README.md

```diff
@@ -65,6 +65,23 @@ This option is useful for Windows users where they are restricted to maximum dir
 to maven during the Checkstyle regression run.  For example, if you want to skip site generation, you
 would add `--extraMvnRegressionOptions "-Dmaven.site.skip=true"` to `diff.groovy` execution.
 
+**failsOnError** (f) - this option tells maven to fail if a checkstyle error is encountered during
+report generation. This means that your configuration uses the `<property name="severity" value="warning"/>
+` property.  If Checkstyle reports a violation for any of the projects that are being tested, the report
+generation will fail.  This option is good for usage in Checkstyle CI for no-error testing
+(optional, default is false).
+
+**allowExcludes** (g) - this option tells `diff.groovy` to allow paths and files defined in a
+`projects*.properties` file to be excluded from report generation (optional, default is false).
+
+**checkstyleVersion** (cv) - used to set the Checkstyle version to be used by maven during report
+generation; this option should mainly be used by satiellite projects
+(optional, default is the latest snapshot).
+
+**sevntuVersion** (sv) - used to set the Sevntu-Checkstyle version to be used by maven during
```

Comment:
Same here. We are not saying this supports sevntu regression, right?

Reply:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/828/files#r599509434

Reply:
Same here.

Reply:
> Same here.

What is this comment referring to?

Reply:
This option has been removed now.